internal/plans/changes.go

```diff
@@ -364,7 +364,7 @@ func (rc *ResourceInstanceChange) Simplify(destroying bool) *ResourceInstanceCha
 					GeneratedConfig: rc.GeneratedConfig,
 				},
 			}
-		case CreateThenDelete, DeleteThenCreate:
+		case CreateThenDelete, DeleteThenCreate, CreateAndForget:
```

Comment:
The whole idea of this method gives me headaches so I cannot say if this is ok or not 😞